analytics/config/config.go

```diff
@@ -43,15 +44,28 @@ func NewPBSAnalytics(analytics *config.Analytics) analytics.PBSAnalyticsModule {
 // Collection of all the correctly configured analytics modules - implements the PBSAnalyticsModule interface
 type enabledAnalytics []analytics.PBSAnalyticsModule
 
+// Returns the name of the analytics module
+func (ea enabledAnalytics) GetName() string {
```

Comment:
`enabledAnalytics` is an array of analytics modules. Does it need to satisfy the analytics module interface itself? I fear that it could cause some confusion if the array looks the same as an instance, people may be unsure if they are seeing activity on the full array of analytics modules, or code that is handling a specific instance of an analytics module. It may be better to export the `enabledAnalytics` type if it is needed outside this package.